### ISSUE TITLE
ACCESS-CM2-tasmax.yaml fails cleaning validation with missing timesteps

### ISSUE THREAD

**Issue by brews** [ClimateImpactLab/downscaleCMIP6#348]
From workflow https://argo.cildc6.org/workflows/default/e2e-tasmax-jobs-tv4q6. Blocking progress on run #211.

Both ssp126 and ssp585 failed cleaning validation.

From log of errored validation from ssp126:
```
Validating gs://clean-b1dbca25/cmip6/ScenarioMIP/CSIRO-ARCCSS/ACCESS-CM2/ssp126/r1i1p1f1/day/tasmax/gn/v20210317.zar
r
Traceback (most recent call last):
  File "/argo/staging/script", line 18, in <module>
    _test_timesteps(ds, data_type, time_period)
  File "/opt/dodola/dodola/core.py", line 645, in _test_timesteps
    assert (
AssertionError: projection cmip6 file is missing timesteps, only has 108405
```

ssp585 gives the same error:
```
Validating gs://clean-b1dbca25/cmip6/ScenarioMIP/CSIRO-ARCCSS/ACCESS-CM2/ssp585/r1i1p1f1/day/tasmax/gn/v20210317.zar
r
Traceback (most recent call last):
  File "/argo/staging/script", line 18, in <module>
    _test_timesteps(ds, data_type, time_period)
  File "/opt/dodola/dodola/core.py", line 645, in _test_timesteps
    assert (
AssertionError: projection cmip6 file is missing timesteps, only has 108405
```

**Comment by brews** [comment]
Similar problem to https://github.com/ClimateImpactLab/downscaleCMIP6/issues/345

**Comment by brews** [comment]
Nudging @dgergel

**Comment by brews** [comment]
Closed with PR #365
